plugin/session_view.py

```diff
@@ -127,7 +127,15 @@ def present_diagnostics_async(self, flags: int) -> None:
                 self.view.erase_regions(key)
             elif ((severity <= userprefs().show_diagnostics_severity_level) and
                     (data.icon or flags != (sublime.DRAW_NO_FILL | sublime.DRAW_NO_OUTLINE))):
-                self.view.add_regions(key, data.regions, data.scope, data.icon, flags)
+
+                def handle_same_regions(region: sublime.Region) -> sublime.Region:
```

Comment:
There is a flag called `sublime.DRAW_EMPTY_AS_OVERWRITE` that could be used instead.

Reply:
Or `sublime.DRAW_EMPTY`